### DISCUSSION TITLE
Unlock the Full Power of Ruby 3 With Pattern Matching!

### DISCUSSION THREAD

**Discussion by matugm** [therubyradar/submissions#150] [Submissions]
Hello!

I just learned about Ruby Radar & I'd like to share this video about Pattern Matching in Ruby:

[https://www.youtube.com/watch?v=n9kDvCKSD0I](https://www.youtube.com/watch?v=n9kDvCKSD0I)

It's a pretty cool feature :)

**Comment by cjilbert504** [comment]
@matugm - Thank you so much for the submission! Thanks for all of the great articles and videos you've created over the years!
